advanced-movie-list/.gitignore

```diff
@@ -0,0 +1,18 @@
+# See http://help.github.com/ignore-files/ for more about ignoring files.
```

Comment:
looks like you created a project inside of your project!

You can probably remove anything under the `advanced-movie-list` folder in here, seems like it is just duplicate info you haven't touched.